### DISCUSSION TITLE
Unable to connect - HTTPSConnectionPool

### DISCUSSION THREAD

**Discussion by b101marc** [Taxel/PlexTraktSync#647] [Q&A] [answered]
### The problem

installed via pipe and was running fine. Running behind traefik and worked after initial install but not after a reboot since the internal url changed from 172.17.0.24 to 172.17.0.13.

I've attempted to uninstall plextraktsync and reinstalled however it doesn't give me the option to log back in to my plex.

Able to refresh connection somehow?

### Expected behavior

plextratk sync connects to server and begins working.

### Install method

pipx (Recommended)

### Version

0.17.1

### Python Version

3.9

### Operating System and Version

Ubuntu 18.04

**Comment by glensc** [comment, marked as answer]
use `plex-login` command or delete `.env` file

**Comment by JayMcNasty** [comment]
For what it's worth, I ran into a similar issue when I was messing with my home networking.  The IP address of my server changed.  And the application no longer worked.

I tried everything, including uninstalling plextraktsync and reinstalling.  Nothing worked.

Luckily I had an old screenshot of some of my router settings and I was able to figure out what the server's IP address was at the time I initially set up plextraktsync.

I simply set my server up to use that IP address in my router as a static IP.  Once I reconnected the NAS with the right IP address, the application worked flawlessly again.

Hopefully this helps someone.  You might be SOL if you can't recall the old IP address like I was able to... but hey...

**Reply by glensc** [reply]
`plex-login` command will print your plex servers and their ip addresses
